draft-ietf-quic-recovery.md

```diff
@@ -142,7 +142,7 @@ of frames contained in a packet affect recovery and congestion control logic:
 
 * Packets that contain only ACK frames do not count toward congestion control
   limits and are not considered in-flight. Note that this means PADDING frames
-  cause packets to contribute toward bytes in flight without directly causing an
+  cause packets to contribute toward bytes in-flight without directly causing an
   acknowledgment to be sent.
```

Comment:
I think it was agreed to spell this "bytes in flight" but I could be wrong.

Reply:
If `in-flight` is used, I think other text needs to updated also. The motivation for not using hyphen was the confusion with `bytes_in_flight` as a variable name.

Reply:
"in-flight" when used as a compound modifier (in-flight packets, for example) should usually have a hyphen before the noun.  Here, it's after the noun, so this wouldn't get one.  See [here](https://www.quickanddirtytips.com/education/grammar/how-to-use-a-hyphen), for example.